api/src/test/java/org/pmiops/workbench/workspaces/WorkspacesControllerTest.java

```diff
@@ -236,7 +236,7 @@
           .domainId("Measurement")
           .countValue(256L)
           .prevalence(0.4F)
-          .conceptSynonyms(new ArrayList<String>());
+          .conceptSynonyms(new ArrayList<>());
```

Comment:
Why did we remove <String>?

Reply:
I have an auto-linter configured which does small fixes like this when I touch a file.